### PR TITLE
Update issue templates

### LINKED ISSUE #447: Add template for issues
Add template for issues. 
This is necessary for a convenient description of new tasks or bugs. 

You can use this as a basis:
https://github.com/hexlet-rus/runit/tree/main/.github/ISSUE_TEMPLATE

image

### PR DESCRIPTION
**Pull request by Rishav1707**
## This PR closes the following issue:
closes #447

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/custom.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+name: Custom issue template
```
Comment (fey) [inline .github/ISSUE_TEMPLATE/custom.md:2]: Этот файл лишний

Comment (Rishav1707) [reply .github/ISSUE_TEMPLATE/custom.md:2]: Средства?

Comment (Rishav1707) [reply .github/ISSUE_TEMPLATE/custom.md:2]: Means?

Comment (fey) [reply .github/ISSUE_TEMPLATE/custom.md:2]: Please remove this file, Github allows to create custom issue without template)

Comment (Rishav1707) [reply .github/ISSUE_TEMPLATE/custom.md]: Done👍🏻

**Comment by fey** [comment]
@Rishav1707 can you copy templates from this project: https://github.com/Hexlet/hexlet-sicp/tree/main/.github/ISSUE_TEMPLATE ?
Github has new feature for creating forms for issues - multiples inputs with selects and textareas.

User can select defined values (OS, browsers) and input some text

**Comment by Rishav1707** [comment]
> @Rishav1707 can you copy templates from this project: https://github.com/Hexlet/hexlet-sicp/tree/main/.github/ISSUE_TEMPLATE ? Github has new feature for creating forms for issues - multiples inputs with selects and textareas.
> 
> User can select defined values (OS, browsers) and input some text

Done👍🏻

**Pull request merged**